skrub/_reporting/tests/test_patch_display.py

```diff
@@ -8,88 +8,90 @@
 @pytest.mark.parametrize("repeat_patch", [1, 2])
 @pytest.mark.parametrize("repeat_unpatch", [1, 2])
 def test_patch_display(df_module, repeat_patch, repeat_unpatch, capsys):
-    df = df_module.make_dataframe(
-        dict(
-            a=[1, 2, 3, 4],
-            b=["one", "two", "three", "four"],
-            c=[11.1, 11.2, 11.3, 11.4],
+    with pytest.raises(DeprecationWarning):
```

Comment:
`DeprecationWarning` will be hidden to the end-user (it is only for the developer). Since a user could have used  `patch_display`, a `FutureWarning` is more appropriate.